subsys/net/ip/net_shell.c

```diff
@@ -6045,6 +6152,11 @@ SHELL_STATIC_SUBCMD_SET_CREATE(net_commands,
 	SHELL_CMD(websocket, NULL, "Print information about WebSocket "
 								"connections.",
 		  cmd_net_websocket),
+
+	#if defined(CONFIG_NET_DHCPV4)
+	SHELL_CMD(dhcp, &net_cmd_dhcp, "Show DHCP information.", cmd_net_dhcp),
```

Comment:
I'd rename the command to `dhcpv4` to avoid confustion, there's no support for DHCPv6 yet.

Reply:
I've renamed to dhcp4 instead of dhcpv4.